test/system/test_cvp_client_api.py

```diff
@@ -221,6 +221,119 @@ def test_api_user_operations(self):
+        self.assertIn('createdBy', result['data'])
+        self.assertEqual(result['data']['createdBy'], 'cvpadmin')
+        self.assertIn('createdOn', result['data'])
+        self.assertIsNotNone(result['data']['createdOn'])
+        self.assertIn('moduleListSize', result['data'])
+        self.assertEqual(result['data']['moduleListSize'], 1)
+        self.assertIn('moduleList', result['data'])
+        self.assertEqual(result['data']['moduleList'], [{'name': 'image', 'mode': 'rw'}])
+
+        # Check created role
+        new_role_key = result['data']['key'];
+        result = self.api.get_role(new_role_key)
+        self.assertIsNotNone(result)
+        self.assertIn('key', result)
+        self.assertEqual(result['key'], new_role_key)
+        self.assertIn('name', result)
+        self.assertEqual(result['name'], 'test_cvp_role')
+        self.assertIn('description', result)
+        self.assertEqual(result['description'], 'role description')
+        self.assertIn('createdBy', result)
+        self.assertEqual(result['createdBy'], 'cvpadmin')
+        self.assertIn('createdOn', result)
+        self.assertIsNotNone(result['createdOn'])
+        self.assertIn('moduleListSize', result)
+        self.assertEqual(result['moduleListSize'], 1)
+        self.assertIn('moduleList', result)
+        self.assertEqual(result['moduleList'], [{'name': 'image', 'mode': 'rw'}])
+        initial_role_name = result['name']
+        initial_role_description = result['description']
+        initial_role_modules = result['moduleList']
+
+        if initial_role_name == 'test_cvp_role':
+            update_role_name = 'test_cvp_role2'
+        else:
+            update_role_name = 'test_cvp_role'
+
+        if initial_role_description == 'role description':
+            update_role_description = 'updated role description'
+        else:
+            update_role_description = 'role description'
+
+        if initial_role_modules == [{'name': 'image', 'mode': 'rw'}]:
+            update_role_modules = []
+        else:
+            update_role_modules = [{'name': 'image', 'mode': 'rw'}]
+
+        # Test Update Role
+        result = self.api.update_role(new_role_key, update_role_name,
+                                      update_role_description, update_role_modules)
+        self.assertIsNotNone(result)
+        self.assertIn('data', result)
+        self.assertEqual(result['data'], 'success')
+        result = self.api.get_role(new_role_key)
+        self.assertIsNotNone(result)
+        self.assertIn('key', result)
+        self.assertEqual(result['key'], new_role_key)
+        self.assertIn('name', result)
+        self.assertEqual(result['name'], update_role_name)
+        self.assertIn('description', result)
+        self.assertEqual(result['description'], update_role_description)
+        self.assertIn('createdBy', result)
+        self.assertEqual(result['createdBy'], 'cvpadmin')
```

Comment:
Same as above.

Reply:
changed the test to just check that the createdBy is not none.